gui-js/apps/minsky-electron/src/app/managers/WindowManager.ts

```diff
@@ -161,9 +161,20 @@ export class WindowManager {
     onCloseCallback?: (ev : Electron.Event) => void
   ) {
     const { width, height, title, modal = true, backgroundColor } = payload;
+
+    let minWidth = payload.minWidth;
+    if(!minWidth && width) minWidth = width;
+    if(!minWidth) minWidth = 300;
+
+    let minHeight = payload.minHeight;
+    if(!minHeight && width) minHeight = height;
```

Comment:
I think you mean height here, not width.

Reply:
Might this be cleaner to express as let minWidth=payload.minWidth? payload.minWidth: width? width: 300

Reply:
It's usually not recommended to use the ? : syntax twice in-line for readability... I settled on another notation using || though.

Reply:
That is brilliant. Sometimes you've got to love Javascript! Such an expression would be cast to bool in C++.